build.gradle.kts

```diff
@@ -99,5 +95,18 @@ subprojects {
         }
       }
     }
+
+    if (name != "codegen" && pluginManager.hasPlugin("org.jetbrains.kotlin.jvm")) {
+      apply(plugin = "org.jetbrains.dokka")
+      tasks.named<DokkaTask>("dokkaHtml") {
+        outputDirectory.set(rootDir.resolve("docs/1.x"))
+        dokkaSourceSets.configureEach {
+          skipDeprecated.set(true)
+          externalDocumentationLink {
+            url.set(URL("https://square.github.io/okio/2.x/okio/"))
```

Comment:
Note that 1.x isn't an option because the javadoc is broken https://square.github.io/okio/1.x/okio/

Reply:
Yikes. https://github.com/square/okio/issues/775#issuecomment-699266366